prov/efa/test/efa_unit_test_cq.c

```diff
@@ -286,4 +287,122 @@ void test_rdm_cq_read_failed_poll()
 	efa_unit_test_resource_destruct(&resource);
 }
 
+#if HAVE_EFADV_CQ_EX
+static void rdm_cq_read_unknown_peer_ah(int peer_removed) {
```

Comment:
should be `test_impl_rdm_cq_read_unknow_peer_ah`. since it implemented a test.

Reply:
Sure will address.

Reply:
parameter type should be `bool`.